snuba/datasets/entities/discover.py

```diff
@@ -284,6 +284,7 @@ def attempt_map(
                 ]
             ),
         ),
+        ("app_start_type", String(Modifiers(nullable=True))),
```

Comment:
I would add/modify an E2E test in `test_snql_api.py` or `test_snql_sdk_api.py` that ensures this column can be read correctly.